src/components/Toast/Toast.css

```diff
@@ -0,0 +1,59 @@
+:root{
+    --modal-bg:rgba(0,0,0,0.5);
+}
+
+/* Toast Css */
+
+.toast-container-top,
+.toast-container-bottom,
+.toast-dismiss{
+    padding:1rem 1rem;
```

Comment:
you can use :is() instead of writing full names of classes. For more clarity on this you can read this: https://developer.mozilla.org/en-US/docs/Web/CSS/:is